docs/integrations/azure/locations.md

```diff
@@ -13,7 +13,19 @@ or registered with the
 [catalog-import](https://github.com/backstage/backstage/tree/master/plugins/catalog-import)
 plugin.
 
-Using a service principal:
+:::note Note
+
+An Azure DevOps provider is added automatically at startup for
+convenience, so you only need to list it if you want to supply a
+[personalAccessToken](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate),
+a [service principal](https://learn.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/service-principal-managed-identity),
+or a [managed identity](https://learn.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/service-principal-managed-identity)
+
+:::
```

Comment:
I'm not sure I totally understand what this note is trying to tell me. Not sure if it makes sense to have at all to be honest.

Reply:
I'll remove it an move the links to the specific paragraphs about setting up that type of authentication

Reply:
Done